content/_index.md

```diff
@@ -18,9 +18,17 @@ Contrary to the name's appearance, GitBOM is neither [git](/glossary/git) nor an
 {{< /home/section >}}
 
 {{< home/section title="Why?" >}}
-By correlating every software [artifact](/glossary/artifact) to a verifiable, complete, and concise [artifact tree](/glossary/artifact_tree), GitBOM enables run-time detection of potential vulnerabilities, regardless of the depth in a depencency tree from which that vulnerability originated.
+By constructing a complete, concise, and verifiable [artifact tree](/glossary/artifact_tree) for every [software artifact](/glossary/artifact), [GitBOM](/glossary/gitbom) enables:
+- Run-time detection of potential vulnerabilities, regardless of the depth in a dependency tree from which that vulnerability originated
+- Identifying license obligations across both open source and commercial software
```

Comment:
I deleted this section in my previous PRs intentionally.

Reply:
gitbom doesn't aid license identification workflows beyond the incidental ability to correlate LICENSE.txt file hashes throughout a tree -- and that is itself potentially misleading. I would prefer not to invite this debate on the front page.

Reply:
Agreed, not looking to create controversy on the front page, happy to delete.